react/src/pages/dashboard/ParticipantTable.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Chip, IconButton } from '@material-ui/core';
+import MaterialTable, { MTableToolbar } from 'material-table';
+import { Cancel } from '@material-ui/icons';
+import FileCopyIcon from '@material-ui/icons/FileCopy';
```

Comment:
combine import into the destructured import above and put it below the other `@material-ui` import